Client/Frontend/Strings.swift

```diff
@@ -1668,6 +1668,45 @@ extension String {
                     tableName: "Settings",
                     value: "Default Search Engine",
                     comment: "Accessibility label for default search engine setting.")
+                public static let LearnAboutSuggestions = MZLocalizedString(
+                    key: "Settings.Search.Accessibility.LearnAboutSuggestions.v121",
+                    tableName: "Settings",
+                    value: "Learn more about Firefox Suggest",
+                    comment: "Accessibility label for Learn more about Firefox Suggest.")
+            }
+            
+            public struct Suggest {
+                public static let AddressBarSettingsTitle = MZLocalizedString(
+                    key: "Settings.Search.Suggest.AddressBarSetting.Title.v122",
+                    tableName: "Settings",
+                    value: "Address bar - Firefox Suggest",
+                    comment: "Title for address bar Fx Suggest Settings.")
+                public static let SuggestionsFromFirefoxTitle = MZLocalizedString(
+                    key: "Settings.Search.Suggest.SuggestionsFromFirefox.Title.v122",
+                    tableName: "Settings",
+                    value: "Suggestions from Firefox",
```

Comment:
We do want to use the app name variable here, though! (Sorry it's confusing—in this case, we _are_ referring to the browser, not the "Firefox Suggest" brand).